pkg/frontend/querymiddleware/model.proto

```diff
@@ -48,11 +48,11 @@ message SampleStream {
   ];
   repeated cortexpb.Sample samples = 2 [
     (gogoproto.nullable) = false,
-    (gogoproto.jsontag) = "values"
+    (gogoproto.jsontag) = "values,omitempty"
```

Comment:
I have a suspicion this was deliberately left off, though I couldn't figure out why.